### PR TITLE
feat(chains): integrate migaloo chain

### PR DESCRIPTION
**Pull request by Sen-Com**
This PR adds support for the Migaloo Chain.

### CODE REVIEW

File: data/chainData.js [hunk 1]
```diff
@@ -337,7 +357,8 @@ export const prefixToId = {
     "bcna": 12,
     "gravity": 13,
     "stride": 14,
-    "pylo": 15
+    "pylo": 15,
+    "whale": 16,
```
Comment (sontrinh16) [inline data/chainData.js:361]: i think it should be migaloo

Comment (Sen-Com) [reply data/chainData.js:361]: The token ticker is whale, chain name, and address prefix is migaloo

Comment (sontrinh16) [reply data/chainData.js:361]: oh this is for mapping prefix to array id

Comment (sontrinh16) [reply data/chainData.js:361]: so migaloo would do

**Comment by sontrinh16** [comment]
> Added the missing data. Do you mind reviewing again?

yes let me look into it

**Review by sontrinh16 — APPROVED** [review]
LGTM

**Pull request merged**